### ISSUE TITLE
Strange errors on Ubuntu when typing non-existing command

### ISSUE THREAD

**Issue by jancborchardt** [zeit/hyper#1209]
- **OS version and name**: Ubuntu 16.04
- **Hyper.app version**: 1.0.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: fresh install
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue

Since I can’t figure out how to copypaste in Hyper (all of Ctrl-Shift-C and Ctrl-C and right-click make the focus get lost) here’s a screenshot of the issue.
Whenever I type in an unknown command, I get this long strange and technical apt package issue:
![capture du 2016-12-14 16-32-55](https://cloud.githubusercontent.com/assets/925062/21188268/17696b9e-c21b-11e6-93a9-1d4dce10fdf5.png)
Instead it should simply be something more human like »Command `asdf` not found«, or just the last lines of suggestions with other apps by the OS.

**Comment by ppot** [comment]
this is linux related. Default behavior on OS X is correct in bash., zsh, fish. it's interpeted as 
`zsh: command not found: adadad`

**Comment by radicaled** [comment]
I'm on Ubuntu 16.04 here, pristine Hyper.is installation, cannot reproduce this behavior.

Could be your install, a zsh function called "asdf" for some reason, etc?

**Comment by jancborchardt** [comment]
@radicaled this is reproducible for me with any non-existing string, including `asldfjkfosdfjkhfakufasdkuanekubsaea` which I doubt is a zsh function ;)
Do you have any PPAs added to your system? That seems to be where all the errors come from. The bottom part is fine except all the `\xab\xa0` escaping errors.

**Comment by radicaled** [comment]
@jancborchardt yeah, I have a lot of PPAs, packages, and services installed -- this installation started out as Ubuntu 14.04, and I do all of my dev work on it. :)

**Comment by Pawamoy** [comment]
Same issue here.

![capture du 2016-12-28 14-58-10](https://cloud.githubusercontent.com/assets/3999221/21523436/81cfe9ce-cd0e-11e6-995b-b77951f1f733.png)

On Ubuntu 16.04, Hyper version 1.0.0
